corehq/apps/app_manager/static/app_manager/js/modules/module_view_report.js

```diff
@@ -8,6 +8,7 @@ hqDefine("app_manager/js/modules/module_view_report", function () {
 
         // Hacky: report modules only deal with one kind of multimedia (the menu image/audio),
         // so assume nav_menu_media_specifics has one element.
+        // ToDo: confirm that this isn't broken if reports module also have case search and claim
```

Comment:
Reports modules won't have case search & claim, they don't have case list/detail config.

Reply:
Thank you!